src/index.js

```diff
@@ -1,37 +1,114 @@
 import './style.css';
 
-const taskContainer = document.getElementById('tasks');
-
-const toDoArr = [
-  {
-    desc: 'I will clean my room',
-    bool: 'true',
-    index: 0,
-  },
-  {
-    desc: 'I have to read a book',
-    bool: 'true',
-    index: 1,
-  },
-  {
-    desc: 'Play fifa with Camavinga',
-    bool: 'false',
-    index: 2,
-  },
-];
-
-const getTasks = () => {
-  toDoArr.forEach((activity, index) => {
-    taskContainer.innerHTML += `
-      <li class="activity">
-        <div class="checkBoxex">
-          <input type="checkbox" name="action1" id="${index}"/>
-          <label for="${index}">${activity.desc}</label><br>
+const form = document.getElementById('todoform');
+const todoInput = document.getElementById('newtodo');
+const toDoListEl = document.getElementById('todos-list');
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+let EditTodoId = -1;
+
+const renderTodos = () => {
+  toDoListEl.innerHTML = '';
+
+  todos.forEach((todo, index) => {
+    toDoListEl.innerHTML += `
+        <div class="todo" id="${index}">
+            <i 
+              class="bi ${todo.checked ? 'bi-check-circle-fill' : 'bi-circle'}"
+              style="color : ${todo.color}"
+              data-action="check"
+              ></i>
+            <p class="" data-action="check">${todo.value}</p>
+            <i class="bi bi-pencil-square" data-action="edit"></i>
+            <i class="bi bi-trash" data-action="delete"></i>
         </div>
-        <i class="fa-solid fa-ellipsis-vertical fs-3 "></i>
-      </li>
-    `;
+        `;
   });
 };
 
-getTasks();
\ No newline at end of file
+renderTodos();
+
+// eslint-disable-next-line consistent-return
```

Comment:
- [ ] Please consider keeping the linters enabled in your project as it's generally a good practice. If you come across any errors, I'd recommend trying to resolve them rather than using the disable line. This will help ensure the best quality for your project! 🌟